Source: Ali Polatel <alip@exherbo.org>
Upstream: No
Reason: Decrease the number of simultaneous threads so the test passes under sydbox in a reasonable time.
--- coreutils-8.2/gnulib-tests/test-lock.c	2009-12-12 23:28:24.000000000 +0200
+++ coreutils-8.2/gnulib-tests/test-lock.c	2009-12-12 23:29:28.000000000 +0200
@@ -57,7 +57,7 @@
 #define ENABLE_DEBUGGING 0
 
 /* Number of simultaneous threads.  */
-#define THREAD_COUNT 10
+#define THREAD_COUNT 4
 
 /* Number of operations performed in each thread.
    This is quite high, because with a smaller count, say 5000, we often get
